### PR TITLE
Update docker config files and readme

### LINKED ISSUE #117: Docker environment is still broke
> NOTE I will add more info later :shipit: 

When I ran `$ docker-compose up` the `web-next` service is overwriting the `web` in someway so the web service is not available

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

### PR DESCRIPTION
**Pull request by arielj**
**Description:**

This PR updates the docker files so they work fine and also updates the readme to run either `up web` or `up web-next`.

Closes #117 

___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/pull_request_template.md).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ WORKDIR /app
 # Copy all the Gemfile files
 COPY Gemfile* ./
 
-RUN gem install bundler -v 2.2.26
+RUN gem install bundler -v "$(tail -n 1 Gemfile.lock)"
```
Comment (arielj) [inline Dockerfile:8]: So we use the bundler version from the Gemfile.lock

File: README.md [hunk 2]
```diff
@@ -50,18 +55,25 @@ $ rails spec
...
 
 Run the app
+
 ```
-$ docker-compose up
+$ docker-compose up web
+
+or
+
+$ docker-compose up web-next
```
Comment (arielj) [inline README.md:76]: There rest of the changes are just the code formatter fixing markdown code style

File: docker-compose.yml [hunk 3]
```diff
@@ -2,25 +2,33 @@ version: "3.9"
 services:
   db:
     image: postgres:10.9
+    environment:
+      POSTGRES_PASSWORD: password
+      POSTGRES_USER: postgres
     volumes:
-      - ./tmp/db:/var/lib/postgresql/data
+      - db_data:/var/lib/postgresql/data
```
Comment (arielj) [inline docker-compose.yml:9]: the database data shouldn't be in the tmp folder, it should be safe to delete any content inside ./tmp

**Review by lubc — APPROVED** [review]
Looks good!

**Comment by lubc** [comment]
I can QA this in a moment

**Comment by lubc** [comment]
@arielj Docker setup went well 👍  Thanks!

**Pull request merged**